cinema/views.py

```diff
@@ -46,7 +52,7 @@
 ):
     queryset = Actor.objects.all()
     serializer_class = ActorSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` are commented out. Ensure this is intentional, as it might leave the endpoints unprotected. Consider enabling them for production to secure the API.